### ISSUE TITLE
Exclude alpha or beta versions from notifier

### ISSUE THREAD

**Issue by eboudrant** [JakeWharton/dependency-watch#139]
Is there a way to filter-out preview versions (like `-alpha` or `-beta` or `-rc`), is it possible or is it a feature to add?
Most of the time we use dependency-watch to monitor stable versions.

**Comment by JakeWharton** [comment]
Maybe. Unfortunately there's no hard standard here. I'll reference the Maven pom spec and Gradle version guidelines tomorrow and see what they have to say. Technically the Maven pom spec is all we need, but in practice many popular projects have weird release versions in use.

**Comment by JakeWharton** [comment]
We could always offer a regex filter, but that's a last resort.

**Comment by eboudrant** [comment]
Yes, I was more envisioning a regexp based solution. Ideally I'd like to monitor stable, beta, alpha independently.

**Comment by JakeWharton** [comment]
Can you explain more about the behavior you want with independent monitoring? Does that mean you're running the tool multiple times? Or are you creating multiple repository entries with the same URL and set of coordinates? Or something else?

**Comment by eboudrant** [comment]
Sorry if that wasn't clear, currently I have :
```
[GoogleMaven]
coordinates = [
    "com.android.tools.build:gradle",
    "androidx.compose.compiler:compiler",
]
```
Which give me the latest version, regardless the channel (alpha, beta, stable,...).

With something like that I could separate the channels :
```
[GoogleMaven]
coordinates = [
    "com.android.tools.build:gradle:{^.+?-alpha\d+$}",
    "com.android.tools.build:gradle:{^.+?-beta\d+$}",
    "com.android.tools.build:gradle:{^\d+\.\d+(\.\d+)?$}",
    "androidx.compose.runtime:runtime:{^.+?-beta\d+$}",
    "androidx.compose.runtime:runtime:{^\d+\.\d+(\.\d+)?$}",
]
```
Considering what's in `{` `}` is a regexp.
